server/channels/app/channel.go

```diff
@@ -481,7 +481,7 @@ func (a *App) createDirectChannelWithUser(c request.CTX, user, otherUser *model.
 		}
 
 		if _, err := a.ShareChannel(c, sc); err != nil {
-			return nil, model.NewAppError("CreateDirectChannel", "app.sharedchannel.dm_channel_creation.internal_error", nil, "", http.StatusInternalServerError).Wrap(err)
+			c.Logger().Error("Failed to share newly created direct channel", mlog.String("channelId", channel.Id), mlog.Err(err))
```

Comment:
Nit: not sure of server conventions, but can we use snake case for logging parameters? Makes it easier when consuming Loki.

Reply:
Sure, fixed in https://github.com/mattermost/mattermost/pull/26895/commits/08aa33a6e5642d90fbbe5d693b70bb8efb0b327d